### PR TITLE
[POWERBIDEDICATED] Create a new extension for POWERBIDEDICATED

### PR DESCRIPTION
**Pull request by Juliehzl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@ __pycache__/
 # Distribution / packaging
 .Python
 env/
+e-env/
```
Comment (haroldrandom) [inline .gitignore]: Is it necessary?

File: src/powerbidedicated/azext_powerbidedicated/azext_metadata.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.67",
```
Comment (haroldrandom) [inline src/powerbidedicated/azext_powerbidedicated/azext_metadata.json]: have you tested in 2.0.67? May it be to low?

File: .gitignore (hunk 1 above)
Comment (Juliehzl) [reply .gitignore]: It should have no influence on our code, right? It is my local environment name.

File: src/powerbidedicated/azext_powerbidedicated/azext_metadata.json (hunk 2 above)
Comment (Juliehzl) [reply src/powerbidedicated/azext_powerbidedicated/azext_metadata.json]: There are some breaking changes for 2.0.67 and previous versions including some new features only available after 2.0.67.

**Comment by haroldrandom** [comment]
@Juliehzl Could you give a chance to leverage the new Linter and Extension Release Pipeline?

**Comment by yonzhan** [comment]
add to S167

File: src/index.json [hunk 3]
```diff
@@ -3146,4 +3146,4 @@
         ]
     },
     "formatVersion": "1"
-}
\ No newline at end of file
+}
```
Comment (haroldrandom) [inline src/index.json]:
```suggestion
}
```

remove the newline in the end. It's added by Github.

Comment (haroldrandom) [inline src/index.json]:
```suggestion
}
```
Remove the newline at the end of file. It's added by Github.

**Comment by haroldrandom** [comment]
What's `policy.json`?

**Comment by yonzhan** [comment]
add to S167

**Pull request merged**